README.md

```diff
@@ -0,0 +1,24 @@
+# Docker Devbox with WSL2
+You know the Docker Devbox with Vagrant.
+
+If you want learn more on Docker Devbox, you can read this article: [Docker Devbox](https://github.com/inetum-orleans/docker-devbox)
+
+But now, with WSL2, you can have a Docker Devbox without the overhead of a VM. This is a simple and lightweight setup to get you started.
+
+Less space allowed, less memory used, less CPU used, less overhead.
+
+And you don't even have to install Docker Desktop and therefore without license limitation linked to Docker Desktop.
+
+## Requirements
+WSL2 with Ubuntu
+
+## Installation
+Clone this project and run the following command in the project directory:
```

Comment:
```suggestion
Clone this project inside your WSL and run the following command in the project directory:
```

Reply:
Also put the git clone command here, people are lazy.

Reply:
Corrected